### DISCUSSION TITLE
Image zigbee2mqtt/zigbee2mqtt-aarch64:1.31.2-1 does not exist for addon_45df7312_zigbee2mqtt

### DISCUSSION THREAD

**Discussion by colseven** [Koenkk/zigbee2mqtt#18047] [General]
What can i do? Can't start anything :(
Image zigbee2mqtt/zigbee2mqtt-aarch64:1.31.2-1 does not exist for addon_45df7312_zigbee2mqtt
![bild 2023-06-17 um 16 07 12](https://github.com/Koenkk/zigbee2mqtt/assets/63189272/7f3774d7-3719-4ac0-b258-5e329c47d007)

**Comment by stephane-r** [comment]
Same for me. Maybe try to downgrade version ?

**Comment by goriasha** [comment]
Same here

**Comment by mikkjant** [comment]
I had the same problem. 

For Home Assistant addon check these:
https://www.zigbee2mqtt.io/guide/installation/03_ha_addon.html
https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/issues/502
